src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -30,6 +30,8 @@ class Definition
     private $factoryMethod;
     private $factoryService;
     private $shared = true;
+    private $deprecated = false;
+    private $deprecationTemplate = "The '%service_id%' service is deprecated. You should stop using it, as it will soon be removed.";
```

Comment:
fixed again. My bad, as @javiereguiluz suggested to use `'` instead https://github.com/symfony/symfony/pull/15491#discussion_r39272125, but I will stick to the other cases